lib/client-api/src/story_store.ts

```diff
@@ -569,6 +570,35 @@ export default class StoryStore {
     };
   };
 
+  getStoriesJsonData = () => {
+    const value = this.getDataForManager();
+    const allowed = ['fileName', 'docsOnly', 'framework', '__id'];
```

Comment:
`__isArgsStory`?

Reply:
Do we need to know that for the sidebar?